products.json

```diff
@@ -322,6 +322,44 @@
                         ]
                     }
                 ]
+            },
+            {
+                "text": "Alcohol & Drug Detection",
+                "state": {
+                    "opened": false
+                },
+                "children": [
+                    {
+                        "text": "Software",
+                        "state": {
+                            "opened": false
+                        },
+                        "children": [
+                            {
+                                "text": "Dräger add",
```

Comment:
Isn't the product called "Dräger DrugCheck"?

Reply:
The product will be called "add" while DrugCheck will be the first components in the system. Right now, there is another part in development and one other in planning.